modules/conversion-center/classes/render/core-render.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+namespace Elementor\Modules\ConversionCenter\Classes\Render;
+
+use Elementor\Utils;
+
+/**
+ * Class Core_Render.
+ *
+ * This class handles the rendering of the Link In Bio widget for the core version.
+ *
+ * @since 3.23.0
+ */
+class Core_Render extends Render_Base {
+
+	public function render( \Elementor\Modules\ConversionCenter\Widgets\Link_In_Bio $widget ): void {
```

Comment:
This method is very long. Can you please extract it into smaller functions?

Reply:
@louiswol94 yes, when @willhowat finishes the task I will refactor the method to use smaller methods. The idea is to group the methods by groups like "Social Icons" and "Cta Links" that can be probably reused when we do the other variations and could be put in the base class.

Reply:
@nicoladj77 Any chance we can get it extracted now already? This current render method is a bit messy and does not follow easy.

Reply:
@louiswol94 let me talk with @willhowat because the risk is that it would create a lot of conflicts as he is actively working on this part. So I will have him commit what he has, then refactor, then he can resume work, and I will commit the work here